Tools/scripts/generate_stdlib_module_names.py

```diff
@@ -117,16 +118,11 @@ def list_modules_setup_extensions(names):
 # List frozen modules of the PyImport_FrozenModules list (Python/frozen.c).
 # Use the "./Programs/_testembed list_frozen" command.
 def list_frozen(names):
-    args = [TEST_EMBED, 'list_frozen']
```

Comment:
Since `TEST_EMBED` is not used anymore, line 15 (`TEST_EMBED = os.path.join(SRC_DIR, 'Programs', '_testembed')`) needs to be removed as well.

Reply:
@arhadthedev Thanks!